OneDrive-Cloud-Player/ViewModels/VideoPlayerPageViewModel.cs

```diff
@@ -283,12 +310,29 @@ private async Task<string> RetrieveDownloadURLMedia(MediaWrapper mediaWrapper)
         /// <summary>
         /// Plays the media.
         /// </summary>
+        /// <param name="startTime"></param>
+        /// <returns></returns>
         private async Task PlayMedia(long startTime = 0)
         {
             CheckPreviousNextMediaInList();
 
-            string mediaDownloadURL = await RetrieveDownloadURLMedia(MediaWrapper);
+            FileName = Path.GetFileNameWithoutExtension(MediaWrapper.CachedDriveItem.Name);
+
+            CoreDispatcher dispatcher = CoreWindow.GetForCurrentThread().Dispatcher;
+            
+            FileNameOverlayVisiblity = Visibility.Visible;
 
+            fileNameOverlayTimer.Interval = 5000;
```

Comment:
The interval could be an adjustable usersetting in the future. But as of now the settings-page branch isn't merged yet with the master branch.